### PR TITLE
Feature/add delete contribution

### PR DESCRIPTION
**Pull request by alexkeating**
## Linear Ticket

https://linear.app/govrn/issue/PRO-456/bug-fix-delete-minted-contribution-button

## Description

- Show delete button for minted contributions

### CODE REVIEW

**Review by amrro — APPROVED** [review]
LGTM.

File: apps/protocol-frontend/src/components/DeleteContributionDialog.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ const DeleteContributionDialog = (props: {
 
   const { onClose } = useDisclosure();
   const cancelRef = useRef<HTMLButtonElement>(null);
+  const [deleteing, setDeleting] = useState(false);
```
Comment (jonathanprozzi) [inline apps/protocol-frontend/src/components/DeleteContributionDialog.tsx:23]: minor typo here that may cause confusion later -- `deleting, setDeleting` -- extra 'e' added to `deleting`

File: apps/protocol-frontend/src/components/DeleteContributionDialog.tsx [hunk 2]
```diff
@@ -57,12 +60,9 @@ const DeleteContributionDialog = (props: {
...
               _hover={{ bgColor: 'brand.primary.200' }}
               ml={3}
-
-              onClick={() =>
-                onDelete(!dialog.onConfirm, dialog.contributionId)
-              
-              }
+              onClick={() => onDelete(!dialog.onConfirm, dialog.contributionId)}
               data-testid="deleteContributionConfirm-test"
+              isLoading={deleteing}
```
Comment (jonathanprozzi) [inline apps/protocol-frontend/src/components/DeleteContributionDialog.tsx:65]: if we change to `deleting` we'll want to update this too

**Review by jonathanprozzi — APPROVED** [review]
Minor typo -- we may want to fix incase it throws off our searches for `deleting` in the future, but tested and this is working on Goerli (minted -> deleted)

**Pull request merged**